dpppt-backend-sdk/dpppt-backend-sdk-model/src/main/proto/TemporaryExposureKeyFormatV2.proto

```diff
@@ -0,0 +1,100 @@
+syntax = "proto2";
+package org.dpppt.backend.sdk.model.gaen.proto.v2;
+option java_package = "org.dpppt.backend.sdk.model.gaen.proto.v2";
+
+message TemporaryExposureKeyExport {
+    // Time window of keys in the file, based on arrival
+    // at the server, in UTC seconds.
+    optional fixed64 start_timestamp = 1;
+    optional fixed64 end_timestamp = 2;
+
+    // The region from which these keys came
+    optional string region = 3;
+
+    // Reserved for future use. Both batch_num and batch_size
+    // must be set to a value of 1.
+    optional int32 batch_num = 4;
+    optional int32 batch_size = 5;
+
+    // Information about associated signatures
+    repeated SignatureInfo signature_infos = 6;
+
+    // Exposure keys that are new.
+    repeated TemporaryExposureKey keys = 7;
+
+    // Keys that have changed status from previous key archives,
+    // including keys that are being revoked.
+    repeated TemporaryExposureKey revised_keys = 8;
+}
+message SignatureInfo {
+    // The first two fields have been deprecated.
+//    reserved 1, 2;
+//    reserved "app_bundle_id", "android_package";
+    optional string app_bundle_id = 1;
+    optional string android_package = 2;
```

Comment:
According to https://developers.google.com/android/exposure-notifications/exposure-key-file-format

```suggestion
    reserved 1, 2;
    reserved "app_bundle_id", "android_package";
```